docker/dev_env/Dockerfile

```diff
@@ -60,8 +60,8 @@ RUN bash -c 'chmod -Rv 0777 /opt'
 # Avoid overwriting `.venv`'s from the host
 ENV PDM_VENV_IN_PROJECT="False"
 
-COPY entrypoint.sh /entrypoint.sh
+# Infinite sleep loop so that we create a persistent environment
+# Everything runs in the same container, rather than new one-off containers for each command
+ENTRYPOINT /bin/bash -c 'while true; do sleep 1; done;'
 
-ENTRYPOINT ["/entrypoint.sh"]
-
-CMD ["/bin/sh"]
+CMD ["/bin/bash"]
```

Comment:
We can theoretically drop all of this. The `fedora` image runs continuously with

```
docker run -itd fedora
```

<img width="704" alt="Screenshot 2024-06-15 at 1 54 14 PM" src="https://github.com/WordPress/openverse/assets/16580576/bd04427c-ea64-473b-a9d6-b20fa0b000d1">

Reply:
Oh, good to know! I will remove those entries for sure :+1:

Reply:
Actually, I think I'd rather leave these explicit, both entrypoint and command, because it documents the behaviour we _expect_ without relying on the base image "incidentally" behaving that way. I'd rather explicitly instruct our image to loop and sleep. What do you think?

Reply:
I'll leave that up to you then. I personally would like to avoid instructions that can be omitted without altering the end experience but I can also get behind the value of being explicit.

Reply:
I've pushed it. Part of why I was reluctant is because I felt the need to explain that we _wanted_ that behaviour, so it was documented in the Dockerfile. It's just as many lines as the explicit implementation, so I don't see a difference, really, but I suppose it's nice to use the default configuration as you said. It just doesn't make anything more concise or easier to understand overall, at least not to me.